metabase/metabase
# Bookmark items in collection page

## How to Test

1. Go to a collection page
2. Click on the kebab menu of collection items
3. Click on Bookmark (or Remove Bookmark if already bookmarked)

Bookmark status should change in that item.
Item should be added/removed from bookmarks list on the left sidebar.

<img src=https://user-images.githubusercontent.com/380816/158710488-127c36bc-6dfd-4f7d-96b2-4c44145f15e0.png width=400 />
